script.js

```diff
@@ -5,18 +5,48 @@
 // if at all. If we used the variable SCISSORS everywhere and misspelled
 // it in 1 place, ESLint would help us detect that error before runtime.
 // String constant variable names are usually all uppercase by convention.
+// ---objects---
 var SCISSORS = 'scissors';
 var PAPER = 'paper';
 var STONE = 'stone';
 var REVERSED_SCISSORS = 'reversed scissors';
 var REVERSED_PAPER = 'reversed paper';
 var REVERSED_STONE = 'reversed stone';
-var REPLAY_INSTRUCTIONS =
-  'Now you can type "scissors" "paper" or "stone" to play another round!';
+// ---players---
```

Comment:
@jeremylim91 could we be consistent with our header naming. Some are capitalised and some aren't.